tool/generate_converter

```diff
@@ -7,7 +7,7 @@
 #
 # mkdir /tmp/inputs
 # cp ~/git/dart-sdk/sdk/pkg/_fe_analyzer_shared/lib/src/metadata/* /tmp/inputs
-# rm /tmp/inputs/proto.dart /tmp/inputs/parser.dart /tmp/inputs/scope.dart
+# rm /tmp/inputs/proto.dart /tmp/inputs/parser.dart /tmp/inputs/scope.dart /tmp/inputs/evaluate.dart
```

Comment:
I'm pretty sure this file was added recently and should be excluded

https://github.com/dart-lang/sdk/blob/805935df2102a8506d51c6ff0119a5b0d69074d4/pkg/_fe_analyzer_shared/lib/src/metadata/evaluate.dart#L630